package.json

```diff
@@ -38,7 +38,7 @@
     "start": "nodemon -x 'npm run build:test-gallery' & browser-sync start --server --files 'test-gallery/main.js' --index 'test-gallery/index.html'",
     "poststart": "rm examples/all-examples.json",
     "schema": "typescript-json-schema --required true --noExtraProps true src/spec.ts ExtendedSpec > vega-lite-schema.json",
-    "presite": "tsc && npm run build && bower install && npm run data && npm run build:toc && npm run build:versions",
+    "presite": "tsc && npm run build && npm run data && npm run build:toc && npm run build:versions",
```

Comment:
so now we only rely on NPM?  If so, should we remove all `devDependencies` from bower?